### PR TITLE
Branch ug

### LINKED ISSUE #120: Update UG with income/expense

### PR DESCRIPTION
**Pull request by NeoMinWei**
fixes #120

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -250,13 +559,37 @@ Existing data will be automatically loaded when the program starts up.
...
+| **Add income**                   | `add income /a AMOUNT /t TYPE [/r DAYS] [/d DESCRIPTION]`  |
+| **Add expense**                  | `add expense /a AMOUNT /t TYPE [/r DAYS] [/d DESCRIPTION]` |
+| **Delete cashflow**              | `delete INDEX [/r]`                                        |
+| **Delete income**                | `delete income INDEX [/r]`                                 |
+| **Delete expense**               | `delete expense INDEX [/r]`                                |
+| **Delete recurrence**            | `delete recurrence INDEX [/r]`                             |
+| **list all cashflows**           | `list`                                                     |
+| **list all incomes**             | `list income`                                              |
+| **list all expenses**            | `list expense`                                             |
+| **list all recurring cashflows** | `list recurrence*                                          |
```
Comment (ryan1604) [inline docs/UserGuide.md]: `list recurrence`**

File: docs/UserGuide.md [hunk 2]
```diff
@@ -34,11 +46,308 @@ you a one-stop interface to access a plethora of features to manage your finance
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of `Financial Planner` from [here](http://link.to/duke).
+3. Copy the file to the folder you want to use as the *home folder* for Neo Chatbot.
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar ip.jar` command to run the application.
+5. Refer to the **Features** section below for details of each command.
```
Comment (ryan1604) [inline docs/UserGuide.md]: Download**, neo chatbot?

**Review by ryan1604 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -250,13 +559,37 @@ Existing data will be automatically loaded when the program starts up.
...
-|----------------------|---------------------------|
-| **Set budget**       | `budget set /b BUDGET`    |
-| **Update budget**    | `budget update /b BUDGET` |
-| **Reset budget**     | `budget reset`            |
-| **Delete budget**    | `budget delete`           |
-| **View budget**      | `budget view`             |
-| **Display Overview** | `overview`                |
-| **View balance**     | `balance`                 |
-| **Exit program**     | `exit`                    |
+| Action                           | Format                                                    |
```
Comment (wwweert123) [inline docs/UserGuide.md:562]: Looks good

File: docs/UserGuide.md [hunk 4]
```diff
@@ -34,11 +46,308 @@ you a one-stop interface to access a plethora of features to manage your finance
...
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Download the latest version of `Financial Planner` from [here](http://link.to/duke).
+3. Copy the file to the folder you want to use as the *home folder* for Financial Planner.
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar ip.jar` command to run the application.
+5. Refer to the **Features** section below for details of each command.
 
 ## Features
 
 {Give detailed description of each feature}
+### Notes about the command format
```
Comment (wwweert123) [inline docs/UserGuide.md:57]: Thanks for this

File: docs/UserGuide.md [hunk 5]
```diff
@@ -34,11 +46,308 @@ you a one-stop interface to access a plethora of features to manage your finance
...
+- Items in square brackets are optional.
+
+  e.g. `[/r DAYS]` can be used as `/r 30` or left empty.
+
+### Add cashflow
+
+#### Add income: `add income`
+Adds an income source to the Financial Planner.
+
+Format: `add income /a AMOUNT /t TYPE [/r DAYS] [/d DESCRIPTION]`
```
Comment (wwweert123) [inline docs/UserGuide.md:70]: Wow! very comprehensive

File: src/main/java/seedu/financialplanner/cashflow/Cashflow.java [hunk 6]
```diff
@@ -77,7 +77,8 @@ public String toString() {
 
         if (recur != 0) {
             string += System.lineSeparator() + "   Recurring every: " + recur + " days";
-            string += ", starting from: " + date.format(DateTimeFormatter.ofPattern("MMM dd yyyy"));
+            string += ", date added: " + date.format(DateTimeFormatter.ofPattern("MMM dd yyyy"));
```
Comment (wwweert123) [inline src/main/java/seedu/financialplanner/cashflow/Cashflow.java:80]: 👍

**Review by wwweert123 — APPROVED** [review]
Very nice!

**Pull request merged**